pipelines/introduction-to-generic-pipelines/README.md

```diff
@@ -0,0 +1,273 @@
+
+    > It is considered good pratice to specify paths that are relative to the notebook or script location.
+
+1. Close the node's properties view.
+
+1. Select the `load_data` node and attach a comment to it.
+
+   ![Add node comment](doc/images/add-comment.png)
+
+   The comment is automatically attached to the node.
+
+1. In the comment node enter a descriptive text, such as `Download the data`.
+
+   ![Add node comment](doc/images/add-comment-text.png)
+
+Next, you'll add a data pre-processing notebook to the pipeline and connect it with the first notebook in such a way that it is executed _after_ the first notebook. This notebook cleans the data in  `data/noaa-weather-data-jfk-airport/jfk_weather.csv`, which `load_data` produced.
+
+### Add a second file to the pipeline
+
+Earlier in this tutorial you added a (notebook or Python script) file component to the canvas using the palette. You can also add Jupyter notebooks, Python scripts, or R scripts to the canvas by drag and drop from the JupyterLab File Browser.
+
+1. From the JupyterLab File Browser drag and drop the `Part 1 - Data Cleaning.ipynb` notebook onto the canvas.
+
+   ![Add file from file browser](doc/images/create-node-from-file-browser.png)
+
+1. Customize the file's execution properties as follows:
+   - Runtime image: `Pandas`
+   - Output files: `data/noaa-weather-data-jfk-airport/jfk_weather_cleaned.csv`
+
+1. Attach a comment to the `Part 1 - Data Cleaning` node and enter a description, such as `Clean the data`.
+
+1. Connect the _output port_ of the `load_data` node to the _input port_ of the `Part 1 - Data Cleaning` node to establish a depency between the two notebooks.
+
+   ![Connect two notebook nodes](doc/images/connect-two-nodes.png)
+
+1. Save the pipeline.
+
+    ![Save pipeline](doc/images/save-wip-pipeline.png)
+
+You are ready to run the pipeline.
+
+### Run a generic pipeline from the pipeline editor
+
+When you run a pipeline locally the files are executed on the machine where JupyterLab is running.
+
+1. Click `Run pipeline`.
+
+   ![Run pipeline](doc/images/run-pipeline.png)
+
+1. Accept the default values in the run dialog and start the run.
+
+   ![Run pipeline locally](doc/images/run-pipeline-locally.png)
+
+1. Monitor the pipeline run in the JupyterLab console.
+
+   ![Monitor pipeline run](doc/images/monitor-pipeline-run.png)
+
+   A message similar to the following is displayed in the pipeline editor window after the run completed.
+
+   ![Run pipeline locally](doc/images/pipeline-run-completed.png) 
+
+### Inspect the pipeline run results
+
+A _local pipeline run_ produces the following output artifacts:
+- Each executed notebook is updated and includes the run results in the output cells.
+- If any notebook persists data/files they are stored in the local file system.
+
+You can access output artifacts from the _File Browser_. In the screen capture below the pipeline output artifacts are highlighted in green.
+
```

Comment:
Hmmmm, for years I've been calling this green. But since I am partially color blind I changed the color to something bright and removed the text reference to just state `are highlighted`.